test/dns/dns_api_test.rb

```diff
@@ -223,4 +234,36 @@ def test_delete_returns_error_on_invalid_type
     delete "/test.com/INVALID"
     assert_equal 400, last_response.status
   end
+
+  def test_validate_srv_value!
+    app = Proxy::Dns::Api.new!
+    assert_raise Proxy::Dns::Error do
+      app.validate_srv_value!('0 5 5060 sipserver.example.com. 1 2 3')
+    end
+    assert_raise Proxy::Dns::Error do
+      app.validate_srv_value!('a 5 5060 sipserver.example.com.')
+    end
+    assert_raise Proxy::Dns::Error do
+      app.validate_srv_value!('0 5 sipserver.example.com.')
+    end
+  end
```

Comment:
Coud you either split this test into three or add comments about why exceptions are expected to be raised? The former is a better way to go about it.